src/sonic_ax_impl/lib/quaggaclient.py

```diff
@@ -88,10 +89,9 @@ def bgp_peer_tuple(dic):
 class QuaggaClient:
     HOST = '127.0.0.1'
     PORT = 2605
-    PROMPT_PASSWORD = b'Password: '
+    PROMPT_PASSWORD = b'\x1fPassword: '
```

Comment:
Can you please briefly explain the need to add the `\x1f` byte prefix?

Reply:
It's observation of Quagga and FRR implementations.

---
In reply to: [389239735](https://github.com/Azure/sonic-snmpagent/pull/124#discussion_r389239735) [](ancestors = 389239735)

Reply:
Just curious why it wasn't necessary before?

Reply:
It is not a bug fix, but an improvement in the parser.

---
In reply to: [389307626](https://github.com/Azure/sonic-snmpagent/pull/124#discussion_r389307626) [](ancestors = 389307626)